qipr_approver/approver/forms/about_you_form.py

```diff
@@ -10,11 +10,13 @@ def __init__(self, user=User()):
                           'value': user.username or ''}
 
         self.first_name = {'name': 'first_name',
+                           'placeholder': 'Jane',
```

Comment:
Is it confusing having an example of a name instead of writing out "First Name" or something? I suppose the label is already there. Yeah examples make sense. Maybe we want an "Ex: Jane" or a "I.E. Jane". Seems like an Amber question.